plugins/tailor.cpp

```diff
@@ -109,6 +110,7 @@ static const MatType
 static const std::list<MatType> all_materials = { M_SILK, M_CLOTH, M_YARN, M_LEATHER, M_ADAMANTINE };
 static const std::list<MatType> default_materials = { M_SILK, M_CLOTH, M_YARN, M_LEATHER }; // adamantine not included by default
 static std::list<MatType> material_order = default_materials;
+static bool confiscate = true;
```

Comment:
I think it's better to not cache the value in a local static and instead pull the value from the source of truth when needed with `config.get_bool(CONFIG_CONFISCATE)`. Then we don't need to worry about the local copy getting out of sync

Reply:
any instance-specific persistent data should be stored as a (regular) member of `class Tailor`. see, for example, the `inventory_sanity_checking` control parameter (which controls an undocumented debugging feature). the only global static in the tool should be the `std::unique_ptr<Tailor> tailor_instance`

as an aside, `static std::list<MatType> material_order = default_materials;` (line 111 or thereabouts) should be made a nonstatic member of `class Tailor` rather than a global static